packages/neutrino-middleware-eslint/index.js

```diff
@@ -15,8 +15,7 @@ const reduceToTrueKeys = reduce((acc, key) => assoc(key, true, acc), {});
 const getEslintRcConfig = pipe(
   getEslintOptions,
   clone,
-  assoc('useEslintrc', true),
-  omit(['failOnError', 'emitWarning', 'emitError']),
+  omit(['failOnError', 'emitWarning', 'emitError', 'cwd', 'useEslintrc', 'fix', 'extensions']),
```

Comment:
I wonder if we should add a comment explaining why some keys are omitted and linking to:
https://github.com/eslint/eslint/blob/master/conf/config-schema.js
(Used by: https://github.com/eslint/eslint/blob/9d1df92628dd4dd1e70fbb19454008e146387435/lib/config/config-validator.js#L167 )